docs/data/data-grid/export/export.md

```diff
@@ -135,7 +137,7 @@ You can provide a [`valueFormatter`](/x/react-data-grid/column-definition/#value
 ### File encoding
 
 You can use `csvOptions` to specify the format of the export, such as the `delimiter` character used to separate fields, the `fileName`, or `utf8WithBom` to prefix the exported file with UTF-8 Byte Order Mark (BOM).
-For more details on these options, please visit the [`csvOptions` api page](/x/api/data-grid/grid-csv-export-options/).
+For more details on these options, please visit the [`csvOptions` API page](/x/api/data-grid/grid-csv-export-options/).
```

Comment:
Let me think that we could add a rule to automatically detect those inconsistencies:
https://vale.sh/docs/topics/styles/#substitution

If you have other typo in mind you face frequently, add them in comment I will create a first rule with them